components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,40 @@
+import { IComment } from "@/types/comment";
+import Comment from "./Comment";
+
+import Image from "next/image";
+import replayEmptyImage from "@/assets/images/img_reply_empty.png";
+
+interface CommentListProps {
+  commentList: IComment[];
+}
+
+function CommentList({ commentList: comments }: CommentListProps) {
+  return (
+    <div>
+      {!!comments.length ? (
+        <ul className="comment-lists">
+          {comments.map((comment: IComment) => (
+            <div key={comment.id}>
+              <Comment comment={comment} />
+            </div>
```

Comment:
`ul`태그를 썼으니 `li` 태그를 사용해주세요. 

```suggestion
            <li key={comment.id}>
              <Comment comment={comment} />
            </li>
```